doc/doxygen/options.dox.in

```diff
@@ -39,12 +39,17 @@ BUILTIN_STL_SUPPORT    = YES
 # Build related configuration options
 #---------------------------------------------------------------------------
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
 EXTRACT_PRIVATE        = YES
 EXTRACT_STATIC         = YES
 EXTRACT_LOCAL_CLASSES  = YES
-HIDE_UNDOC_MEMBERS     = YES
-HIDE_UNDOC_CLASSES     = YES
+EXTRACT_STATIC         = YES
+EXTRACT_PRIVATE        = YES
+EXTRACT_PRIV_VIRTUAL   = YES
```

Comment:
What's the default on the latter two? A user doesn't benefit from seeing private member functions, and I don't think we should show them in the html pages...

Reply:
all three are NO: http://www.doxygen.nl/manual/config.html#cfg_extract_private

Reply:
I personally don't mind seeing private member function when looking at the documentation but I see your point.

Reply:
I'd advocate for leaving them at `NO`.

Reply:
I am fine with that. Thoughts on "HIDE_UNDOC*" ?

Reply:
We strive to document everything, and what we forget to document should probably still show up. So I'm find with not hiding undocumented functions/classes.